scripts/apps/web-publisher/controllers/WebPublisherManagerController.js

```diff
@@ -29,8 +41,8 @@ export function WebPublisherManagerController($scope, publisher, modal) {
      * @description Opens modal window for creating new site
      */
     self.toogleCreateSite = () => {
```

Comment:
I'm just wondering - why are some functions defined as arrow functions and others not? Is it to bind to `this`? You are already using self here ...

Could potentially be much cleaner as a class...